pkg/util/git.go

```diff
@@ -13,7 +13,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package git
+package util
```

Comment:
curious why we decided to move here in this pkg.. Seems like they could be in git pkg

Reply:
Originally, there were duplicates of the until functions in the git package because of cyclic dependencies.  It was confusing because some of the parser code was using the same named functions in the util package while others were using the ones from the git package.